studio/components/layouts/SignInLayout/SignInLayout.tsx

```diff
@@ -89,7 +90,7 @@ const SignInLayout = ({
       )}
 
       <div className="flex flex-col flex-1 bg-scale-100">
-        <div className="absolute top-0 w-full px-8 pt-6 mx-auto sm:px-6 lg:px-8">
+        <div className={`absolute top-0 w-full px-8 mx-auto sm:px-6 lg:px-8 ${ongoingIncident ? 'pt-16' : 'pt-6'}`}>
```

Comment:
@brenelz any way we can use flexbox or similar layouting techniques to push the page down vs. manually adding padding?

Reply:
I woulda like to do it that way but its tricky because this is absolutely positioned

Reply:
Found the same @brenelz yeah. This is fine as is I think.